apps/examples/st_things/blink/sample_blink_things.c

```diff
@@ -66,10 +66,30 @@ static bool handle_set_request(st_things_set_request_message_s *req_msg, st_thin
 	return false;
 }
 
-int ess_process(void)
+int ess_process(bool stop)
 {
+	if (stop) {
+		st_things_error_e res = st_things_deinitialize();
+		if (res == ST_THINGS_ERROR_NONE) {
+			printf("[%s]=====================================================\n", TAG);
+			printf("[%s]                    Stack Stopped & Deinitialized                    \n", TAG);
+			printf("[%s]=====================================================\n", TAG);
+		} else {
+			printf("[%s]Failed to stop & deinitialize the stack. Error code: %d\n", TAG, res);
+		}
+		return 0;
+	}
+
 	bool easysetup_complete = false;
-	st_things_initialize("device_def.json", &easysetup_complete);
+	st_things_error_e err_code = st_things_initialize("device_def.json", &easysetup_complete);
+	if (err_code == ST_THINGS_ERROR_NONE) {
+		printf("[%s]=====================================================\n", TAG);
+		printf("[%s]                    Stack Initialized                    \n", TAG);
+		printf("[%s]=====================================================\n", TAG);
+	} else {
+		printf("[%s]Failed to initialize. Error code: %d\n", TAG, err_code);
+		return 0;
```

Comment:
please check return value;
this is negative case,
i think Positive and negative values should be different.

Reply:
The caller function doesn't check return value whether is positive or negative and what else.